### PR TITLE
fix to #1294

### PR DESCRIPTION
**Pull request by Argn0**
I'm a total noob. I don't expect these change to get merged but I had fun anyway :D

### CODE REVIEW

File: src/components/Match/MatchStory.jsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ const TeamSpan = isRadiant => (
 
 // Modified version of PlayerThumb
 const PlayerSpan = (player) => {
-  if (!player || !heroes[player.hero_id]) {
+  if (!player || (!player && !heroes[player.hero_id])) {
```
Comment (howardchung) [inline src/components/Match/MatchStory.jsx]: I don't think this change is needed

File: src/components/Match/CrossTable.jsx [hunk 2]
```diff
@@ -16,6 +16,8 @@ import constants from '../constants';
 
 const teamIconStyle = { width: '30px', height: '30px' };
 
+const default_hero = { name: 'Unknown Hero' };
```
Comment (howardchung) [inline src/components/Match/CrossTable.jsx]: Move the english string to langs/en-US.json (don't need to do the other languages manually)

**Comment by howardchung** [comment]
do npm run lint to check for lint issues

File: src/components/Match/MatchStory.jsx (hunk 1 above)
Comment (Argn0) [reply src/components/Match/MatchStory.jsx]: I don't exactly understand what's happening but if I remove this change then the Story tab crashes.

example: matches/3564865624/story

File: src/components/Visualizations/Graph/MatchGraph.jsx [hunk 3]
```diff
@@ -188,7 +188,7 @@ class PlayersGraph extends React.Component {
               {match.players.map((player) => {
                 const hero = heroes[player.hero_id] || {};
                 const playerColor = playerColors[player.player_slot];
-                const isSelected = (hoverHero === heroes[player.hero_id].localized_name);
+                const isSelected = (hoverHero === heroes[player.hero_id] && heroes[player.hero_id].localized_name);
```
Comment (rodrigofepy) [inline src/components/Visualizations/Graph/MatchGraph.jsx]: This line is a string comparison, I think you should move the object validation in front of the comparison.
Should look like this: `heroes[player.hero_id] && (hoverHero === heroes[player.hero_id].localized_name);`

Comment (Argn0) [reply src/components/Visualizations/Graph/MatchGraph.jsx]: thanks. I changed it as you suggested.

**Comment by howardchung** [comment]
combining with https://github.com/odota/ui/pull/1318

**Pull request closed without merging**